R/html.R

```diff
@@ -901,7 +902,8 @@ restore_appendix_html = function(x, remove = TRUE) {
 parse_references = function(x) {
   i = grep('^<div id="refs" class="references[^"]*">$', x)
   if (length(i) != 1) return(list(refs = character(), html = x))
-  r = '^<div id="(ref-[^"]+)">$'
+  regid = ' id="(ref-[^"]+)"'
+  r = sprintf('^<div%s.*>$', regid)
```

Comment:
How about this?

```suggestion
  r = '^(<div) id="(ref-[^"]+)"(.*>)$'
```

Then on line 912, use `  ids = gsub(r, '\\2', x[k])`

Reply:
Oh I see. yes that looks of. Does the same but less lines 😄 

 Re reading it `[^>]*>` should be better than `.*>` also. 

I'll do the change.

Reply:
I think you are correct.